api/v1/verticadb_webhook.go

```diff
@@ -1029,6 +1135,16 @@ func (v *VerticaDB) isDBInitialized() bool {
 	return v.IsStatusConditionTrue(DBInitialized)
 }
 
+// isSandboxInitialized returns ture when the sandbox has been created in the database
+func (v *VerticaDB) isSandboxInitialized(targetSc string) bool {
```

Comment:
I know it is not a big deal
```suggestion
func (v *VerticaDB) isSandboxInitialized(targetSb string) bool {
```